### PR TITLE
Add high level overview of the process and testing

### PR DESCRIPTION
**Pull request by lauraGgit**
give a bit more high level overview about what developers should expect from the integration process

### CODE REVIEW

**Comment by lauraGgit** [comment]
Also addresses #69 / #68

File: _data/nav.yml [hunk 1]
```diff
@@ -1,6 +1,8 @@
 primary:
-  - text: FAQ
-    href: /faq/
+  - text: Getting Started
+    href: /
+  - text: Integration flow
+    href: /overview/
```
Comment (caitlinhumphrey) [inline _data/nav.yml]: This flow makes so much more sense!

File: _pages/overview.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+title: Integration overview
+lead: >
+  Login.gov is an identity provider that integrates with your application using industry protocols.
+---
+
+Login.gov is a FedRAMP moderate approved multifactor authentication and identity proofing platform that makes online interactions with the U.S. government simple, efficient and intuitive.
+
+<h2>Integration flow</h2>
+* Once a [service provider configuration](/#service-provider-configuration) is provided in one of login.gov's environments, users start at your application and are redirected back to login.gov via [OIDC]({{ site.baseurl }}/oidc) or [SAML]({{ site.baseurl }}/saml) protocols.
+* Your application request will determine if the request will be processed as just an authenication request at NIST Identity Assurance Level 1 (IAL1) or as an identity proofed event at NIST Identity Assurance Level 2 (IAL2).
```
Comment (caitlinhumphrey) [inline _pages/overview.md]: please address authentication typo

File: _pages/testing.md [hunk 3]
```diff
@@ -1,21 +1,22 @@
...
+Login.gov provides an open sandbox enviroment to create and test integrations between login.gov and you applications. In the sandbox environment, we provide a dashboard where you can manage your test applications. While the dashboard is open to use, you must request access.
 
-1. First, the login.gov team will create accounts for you and your team members. Once you have an account, head over to the dashboard at [dashboard.int.identitysandbox.gov](https://dashboard.int.identitysandbox.gov)
-2. In the upper-right corner, click **Log in**. You'll be prompted to sign in or create an account with the login.gov IdP in the agency integration environment (hosted at [idp.int.identitysandbox.gov](https://idp.int.identitysandbox.gov)).
+1. Request that accounts your emails get added to the dashboard application in the test sandbox environment. You can request [access here](https://share.hsforms.com/16DIoo--rTU2xbNW1MShkBg3ak9e). The login.gov team will create accounts for you and your team members. Once you have an account, head over to the dashboard at [dashboard.int.identitysandbox.gov](https://dashboard.int.identitysandbox.gov).
```
Comment (caitlinhumphrey) [inline _pages/testing.md]: We'll keep this for now, but will likely change the way developers request sandbox access via email

File: _pages/testing.md [hunk 4]
```diff
@@ -1,21 +1,22 @@
...
+2. In the upper-right corner, click **Log in**. You'll be prompted to sign in or create an account with the test login.gov IdP in the agency integration environment (hosted at [idp.int.identitysandbox.gov](https://idp.int.identitysandbox.gov)). **Please note that if you already have a login.gov account that this is a test environment that is not linked to your production account.**
 3. Once logged in, you'll be sent to the "My apps" page, which will list all the test applications you and your team have created.
-4. Click **Create a new test app** and fill out the form to register a new application with the login.gov IdP in the agency integration environment.
+4. Click **Create a new test app** and fill out the form to register a new application with the login.gov IdP in the test sandbox environment.
 5. Start testing! The login.gov team will help you along the way.
```
Comment (caitlinhumphrey) [inline _pages/testing.md]: Should add text about getting added to the Slack channel, which serves as our community of practice for troubleshooting and getting in touch with our developers.

File: _pages/testing.md (hunk 3 above)
Comment (lauraGgit) [reply _pages/testing.md]: That makes sense

File: _pages/overview.md (hunk 2 above)
Comment (lauraGgit) [reply _pages/overview.md]: good catch!

File: _pages/testing.md (hunk 4 above)
Comment (lauraGgit) [reply _pages/testing.md]: Added.

**Comment by lauraGgit** [comment]
@blacktm @caitlinhumphrey has reviewed but doesn't yet have edit rights- can you approve/merge?

**Pull request merged**